exercises/all-your-base/example.h

```diff
@@ -0,0 +1,13 @@
+#ifndef ALL_YOUR_BASE_H
```

Comment:
Nitpick: this should be `#if !defined(ALL_YOUR_BASE_H)` to match the other exercises